kratos/templates/template_application/custom_python/add_custom_strategies_to_python.cpp.in

```diff
@@ -42,7 +42,7 @@ void  AddCustomStrategiesToPython(pybind11::module& m)
     typedef UblasSpace<double, Matrix, Vector> LocalSpaceType;
 
     typedef LinearSolver<SparseSpaceType, LocalSpaceType > LinearSolverType;
-    typedef SolvingStrategy< SparseSpaceType, LocalSpaceType, LinearSolverType > BaseSolvingStrategyType;
+    typedef ImplicitSolvingStrategy< SparseSpaceType, LocalSpaceType, LinearSolverType > BaseSolvingStrategyType;
```

Comment:
hm guess this should be changed

can you add the second exposure (with name `ImplicitSolvingStrategy`) and deprecate the other one?
For me fine to do it in a separate PR to merge this one soon

Reply:
Makes sense. I can change it in here.

Reply:
Indeed, maybe it makes more sense that we change what is done below to add an example of implicit and explicit strategy export.